File: Q25/Q25.py
rows = int(input("Enter the number of rows: "))

for i in range(1, rows + 1):
    print(' ' * (rows - i), end='')
    for j in range(1, 2 * i):
        if i == rows or j == 1 or j == 2 * i - 1:
            print('*', end='')
        else:
            print(' ', end='')
    print()

File: Q25/Q25.js
function printHollowPyramid(rows) {
    for (let i = 1; i <= rows; i++) {
        let str = ' '.repeat(rows - i);
        for (let j = 0; j < 2 * i - 1; j++) {
            if (i === rows || j === 0 || j === 2 * i - 2)
                str += '*';
            else
                str += ' ';
        }
        console.log(str);
    }
}

const rows = parseInt(prompt("Enter the number of rows: "), 10);
printHollowPyramid(rows);
